templates/projects/settings.html

```diff
@@ -24,7 +24,7 @@
 </div>
 
 <div class="row">
-    <div id="import_tasks" class="col-md-6">
+    <div id="manage_coowners" class="col-md-6">
```

Comment:
There are some more duplicate ids. Do you want to fix them as part of the PR too? I see `import_tasks` and `export_tasks` duplicated above.

Reply:
Fixed via https://github.com/bloomberg/pybossa-default-theme/pull/170/commits/1241b437a6297faf192db8e718af9f7161a6d833
Thanks!